src/common/transformations/src/transformations/common_optimizations/transpose_sinking_utils.cpp

```diff
@@ -71,9 +71,38 @@ void SwapNames(NodePtr node1, NodePtr node2) {
     SwapOutputNames(node1->output(0), node2->output(0));
 }
 
+namespace {
+
+std::set<size_t> GetNodeIds(NodeVector nodes) {
```

Comment:
Please use `const NodeVector&` , check all functions.
I believe it's ok not to use `&` in case of shared_ptr, but it's better to add `const` qualifier too